frontend/src/components/ReviewerCard.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="card mb-3">
+    <div class="card-header">
+      <h5 class="card-title">
+        {{ reviewer.memberProfile.name }} ({{ reviewer.memberProfile.username }})
+      </h5>
+    </div>
+    <div class="card-body">
+      <p class="card-text">
+        <strong>Email:</strong> {{ reviewer.memberProfile.email }}<br />
+        <strong>Programming Languages:</strong> {{ reviewer.programmingLanguages.join(", ") }}<br />
+        <strong>Subject Areas:</strong> {{ reviewer.subjectAreas.join(", ") }}<br />
+        {{ reviewer.notes }}
+      </p>
+      <a class="btn btn-primary me-1" @click="emit('edit')">Edit</a>
+      <a v-if="reviewer.isActive" class="btn btn-danger" @click="changeActiveState(reviewer, false)"
+        >Deactivate</a
+      >
+      <a v-else class="btn btn-success" @click="changeActiveState(reviewer, true)">Activate</a>
```

Comment:
minor aesthetic gripe: the two different greens side by side look off to me and the buttons are a bit large. How about the following, which uses the same styling as the contributor cards in the release editor:

```html
<template>
  <div class="card mb-3">
    <div class="card-header d-flex justify-content-between align-items-center">
      <h5 class="card-title mb-0">
        {{ reviewer.memberProfile.name }} ({{ reviewer.memberProfile.username }})
      </h5>
      <span>
        <a class="btn btn-link" @click="emit('edit')">
          <small><i class="fas fa-edit me-1"></i>Edit</small>
        </a>
        <a
          v-if="reviewer.isActive"
          class="btn btn-link text-danger"
          @click="changeActiveState(reviewer, false)"
        >
          <small><i class="fas fa-trash me-1"></i>Deactivate</small>
        </a>
        <a v-else class="btn btn-link text-success" @click="changeActiveState(reviewer, true)">
          <small><i class="fas fa-check-circle me-1"></i>Activate</small>
        </a>
      </span>
    </div>
    <div class="card-body">
      <p class="card-text">
        <strong>Email:</strong> {{ reviewer.memberProfile.email }}<br />
        <strong>Programming Languages:</strong> {{ reviewer.programmingLanguages.join(", ") }}<br />
        <strong>Subject Areas:</strong> {{ reviewer.subjectAreas.join(", ") }}<br />
        {{ reviewer.notes }}
      </p>
    </div>
  </div>
</template>
```

Reply:
![image](https://github.com/user-attachments/assets/8b597c4b-469a-43d1-8f0f-59ffc45121aa)